app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -115,7 +116,16 @@ ShortFormApplicationController = (
       if options.path
         $state.go(options.path)
     else
-      $scope.hideAlert = false
+      $scope.handleErrorState()
+
+  $scope.handleErrorState = ->
+    # show error alert
+    $scope.hideAlert = false
+    el = angular.element(document.getElementById('short-form-wrapper'))
+    # uses duScroll aka 'angular-scroll' module
+    topOffset = 0
+    duration = 400 # animation speed in ms
+    $document.scrollToElement(el, topOffset, duration)
```

Comment:
@chrisdolendo good call on the `0, 400` not having any context for other devs viewing this code, I updated that to be more clear